cmake/Dependencies.cmake

```diff
@@ -1623,7 +1623,8 @@ if(NOT INTERN_BUILD_MOBILE)
     add_compile_options(-DUSE_GCC_GET_CPUID)
   endif()
 
-  find_package(AVX) # checks AVX and AVX2
+  FIND_PACKAGE(VSX) # checks VSX
```

Comment:
could you keep the lower-case versions and stick to that please?

Reply:
sure

Reply:
ok